pkg/ingress/class_loader_test.go

```diff
@@ -59,7 +60,7 @@ func Test_defaultClassLoader_Load(t *testing.T) {
 					},
 				},
 			},
-			wantErr: errors.New("invalid ingress class: spec.ingressClassName is nil"),
+			wantErr: nil,
```

Comment:
Should specify `want` instead of `wantErr`.

Need to add a test where there is a default IngressClass. It should assert that the default IngressClass was used.

Need to add a test where there are two IngressClasses annotated as being the default.

Reply:
I added multiple test case
1- when IngressClass is ALB - Multiple default classes specified
2- when IngressClassName unspecified - Multiple default classes specified
3- when IngressClassName unspecified - Default class specified 
4- when IngressClassName unspecified - Default class unspecified